addons/account/models/account_move.py

```diff
@@ -4436,7 +4437,7 @@ def fix_remaining_cent(currency, abs_residual, partial_amount):
 
             # Move to the next available debit line.
             if not debit_line:
-                debit_line = next(debit_lines, None)
+                debit_line = next(void_lines, None) or next(debit_lines, None)
```

Comment:
```suggestion
                debit_line = next(debit_lines, None) or next(void_lines, None)
```

I want to match the voids with debits or credits, not with other voids, so that it is part of the chain

Reply:
oh, my bad